include/mfmg/utils.cuh

```diff
@@ -51,6 +51,9 @@ convert_matrix(dealii::TrilinosWrappers::SparseMatrix const &sparse_matrix);
 SparseMatrixDevice<double>
 convert_matrix(Epetra_CrsMatrix const &sparse_matrix);
 
+dealii::TrilinosWrappers::SparseMatrix
+convert_matrix(SparseMatrixDevice<double> const &matrix_dev);
```

Comment:
Same name for conversion `Epetra_CrsMatrix` -> `SparseMatrixDevice` and `SparseMatrixDevice` -> `dealii::TrilinosWrappers::SparseMatrix` is a confusing.